lib/environment.sh

```diff
@@ -86,7 +86,13 @@ write_profile() {
   local bp_dir="$1"
   local build_dir="$2"
   mkdir -p "$build_dir/.profile.d"
-  cp "$bp_dir"/profile/* "$build_dir/.profile.d/"
+  local restore_extglob
+  restore_extglob=$(shopt -p extglob) # will return "shopt -u extglob" (or "-s"), for later
+  shopt -s extglob
+  cp "$bp_dir"/profile/!(WEB_CONCURRENCY.sh) "$build_dir/.profile.d/"
```

Comment:
@dzuelke all the tests seem to be breaking because of this line:

```
SSERT:Expected STD_ERR to be empty; was </tmp/testtCMCD/lib/environment.sh: line 92: syntax error near unexpected token `('> expected:<> but was:</tmp/testtCMCD/lib/environment.sh: line 92: syntax error near unexpected token `('>
```

Reply:
Yes, an shUnit2 bug. Let's see if updating that fixes it.

Reply:
Ah, or it's https://stackoverflow.com/questions/49283740/bash-script-throws-syntax-errors-when-the-extglob-option-is-set-inside-a-subsh

Reply:
Alright, let's just do explicit file names for the `cp` ;)

Reply:
Better. Now the failures are limited to just the concurrency tests 😄 

```
ASSERT:Expected STD_ERR to be empty; was </__w/heroku-buildpack-nodejs/heroku-buildpack-nodejs/profile/WEB_CONCURRENCY.sh: line 35: cgroup_util_read_cgroup_memory_limit_with_fallback: command not found> expected:<> but was:</__w/heroku-buildpack-nodejs/heroku-buildpack-nodejs/profile/WEB_CONCURRENCY.sh: line 35: cgroup_util_read_cgroup_memory_limit_with_fallback: command not found>
shunit2:ERROR testConcurrency1X() returned non-zero return code.
```